### DISCUSSION TITLE
Orleans integration with Process Framework

### DISCUSSION THREAD

**Discussion by Kevdome3000** [microsoft/semantic-kernel#9460] [General]
Wondering if I can help get this started as it something that is critical for my use case.

Seems like I can draft off the progress being made with Dapr runtime integration but would like to know if:
 a) the team is willing to take PRs for this
 b) are there any specific requirements or constraints unique to Orleans that I should be aware of
 
 Thanks

**Comment by sophialagerkranspandey** [comment]
Thanks for reaching out, we currently have Orleans integration in our backlog:[Backlog by Bucket · Semantic Kernel (github.com)](https://github.com/orgs/microsoft/projects/866/views/59). At this time there aren't any requirements or constraints unique to Orleans to be aware of. Thanks!
